### PR TITLE
Add docs for unpacking and adding ScandyCore package.

### PR DESCRIPTION
**Pull request by kaben**

### CODE REVIEW

File: dependencies/README.md [hunk 1]
```diff
@@ -1,2 +1,10 @@
 # dependencies
 Put your ScandyCore package here!
+
+For example, once you've downloaded the ScandyCore package for MacOs, do this:
+```
+$ pushd dependencies/
+$ mv ~/Downloads/ScandyCore-0.0.3+1690.1bec63f-Darwin.zip ./
+$ unzip ScandyCore-0.0.3+1690.1bec63f-Darwin.zip
+$ ln -s ScandyCore-0.0.3+1690.1bec63f-Darwin.zip ScandyCore
```
Comment (hcwiley) [inline dependencies/README.md]: shouldn't this be:

```ln -s ScandyCore-0.0.3+1690.1bec63f-Darwin ScandyCore```

?

Comment (hcwiley) [inline dependencies/README.md]: also can you change to not have `$` in every line. this prevents developers from copy pasting which is frustrating for quick onboarding

File: dependencies/README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
 # dependencies
 Put your ScandyCore package here!
+
+For example, once you've downloaded the ScandyCore package for MacOs, do this:
+```
```
Comment (hcwiley) [inline dependencies/README.md]: you can change this to:
\```bash
and it will know its bash code

**Comment by kaben** [comment]
@hcwiley : Done!

**Comment by hcwiley** [comment]
thanks! looking out for new developer on-boarding. want to avoid this:

```bash
blu-mac:ScandyCoreQtExample hcwiley$ $ mkdir -p build
-bash: $: command not found
blu-mac:ScandyCoreQtExample hcwiley$ $ cd build
-bash: $: command not found
blu-mac:ScandyCoreQtExample hcwiley$ $ cmake ../
-bash: $: command not found
blu-mac:ScandyCoreQtExample hcwiley$ $ make
blu-mac:ScandyCoreQtExample hcwiley$
```

**Pull request merged**